### DISCUSSION TITLE
Add the ability to set a request body

### DISCUSSION THREAD

**Discussion by BrunoBlanes** [grafana/grafana-infinity-datasource#447] [Ideas]
Although the HTTP spec does not specify a body being useful for `GET` requests, I've found quite a few API that use `POST` endpoints with a `body` to pass the query parameters. These endpoints also usually reply to `GET` requests.

I guess this could be either "allow setting a `body` for `GET` requests, or allow `POST` requests with a body.

**Comment by yesoreyeram** [comment]
I don't have plans to support outliers. But feel free to send PR.
